backend/src/bundles/auth/auth.service.ts

```diff
@@ -49,8 +56,21 @@ class AuthService {
             });
         }
 
+        const dataFromLMS =
+            await this.lmsDataService.getUserDataFromLMSServerbyEmail(email);
+        const isUserTalent = role === UserRole.TALENT;
+
+        if (!dataFromLMS && isUserTalent) {
+            throw new HttpError({
+                message: ErrorMessage.NOT_FOUND_ON_LMS,
+                status: HttpCode.BAD_REQUEST,
+            });
+        }
+
         const user = await this.userService.create(userRequestDto);
 
+        await this.lmsDataService.addUserLMSDataToDB(user.id, dataFromLMS);
```

Comment:
As improvement, we can do data fetching from LMS only when role === UserRole.TALENT, because for now we just do unused request for users which have other role

Reply:
really, I will fix it